packages/edit-site/src/components/page-template-parts/dataviews-template-parts.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { TEMPLATE_PART_POST_TYPE } from '../../utils/constants';
+
+import DataviewsTemplatesTemplateParts from '../page-templates/dataviews-templates-template-parts';
+
+export default function DataviewsTemplateParts() {
+	return (
+		<DataviewsTemplatesTemplateParts postType={ TEMPLATE_PART_POST_TYPE } />
```

Comment:
Is this file really needed, why not just use the underlying component directly?

Reply:
For now it's not, but I thought it could when we add the view handling.. I can definitely remove for now.